shared/src/business/useCases/courtIssuedDocument/fileAndServeCourtIssuedDocumentInteractor.test.js

```diff
@@ -496,16 +530,22 @@ describe('fileAndServeCourtIssuedDocumentInteractor', () => {
   });
 
   it('should throw an error if there is no one on the case with electronic or paper service', async () => {
+    const petitioners = [
+      {
+        ...caseRecord.petitioners[0],
+        serviceIndicator: 'None',
+      },
+    ];
+
     applicationContext
       .getPersistenceGateway()
-      .getCaseByDocketNumber.mockReturnValue({
+      .getCaseByDocketNumber.mockReturnValueOnce({
+        ...caseRecord,
+        petitioners,
+      })
+      .mockReturnValueOnce({
         ...caseRecord,
-        petitioners: [
-          {
-            ...caseRecord.petitioners[0],
-            serviceIndicator: 'None',
-          },
-        ],
+        petitioners,
```

Comment:
❓ Same question here about mockReturnValueOnce. We are using it twice to return the same thing, and then not providing a default.

Reply:
I am not quite sure about this one.  This did not originally fail when we first started updating this test.  However, when I commented it out, the test failed because it expected the promise to reject and throw an error.  But, the test instead resolved the promise to a value of undefined.

Reply:
Same response from above applies:
> After talking through this, I believe the reason we are doing this is because the default mock would persist into the later tests and have unintended consequences.

So this test would potentially have been affected by the previous one, and would in turn potentially affect other tests whether they exist yet or not.